<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="gameLogoLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../../../run/user/1000/doc/85f1cdd7/resources.qrc">:/beyond_styx_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="teamLogoLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../../../run/user/1000/doc/85f1cdd7/resources.qrc">:/group_icon_alpha.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="versionComboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <zorder>gameLogoLabel</zorder>
   <zorder>downloadBtn</zorder>
   <zorder>teamLogoLabel</zorder>
   <zorder>versionComboBox</zorder>
   <zorder>refreshBtn</zorder>
   <zorder>playBtn</zorder>
   <zorder>progressBar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../../../run/user/1000/doc/85f1cdd7/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>downloadBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDownloadBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRefreshBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>versionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onVersionIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPlayBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDownloadBtnClick()</slot>
  <slot>onRefreshBtnClick()</slot>
  <slot>onVersionIndexChanged(int)</slot>
  <slot>onPlayBtnClick()</slot>
 </slots>
</ui>
